plugin/src/main/scala/org/ehcache/integrations/play/JCacheModule.scala

```diff
@@ -107,20 +120,25 @@ class CacheManagerProvider @Inject()(env: Environment, config: Configuration, li
 
 private[play] class NamedJCacheProvider(name: String, create: Boolean) extends Provider[Cache[String, Any]] {
   @Inject private var manager: CacheManager = _
-  lazy val get: Cache[String, Any] = NamedJCacheProvider.getNamedCache(name, manager, create).get
+  @Inject private var jCacheWrapper: JCacheWrapper = _
```

Comment:
Been googling, looks like there is no optional inject support. So the way to do a default would be to not use `@Inject` but instead depend on the context to find a bound impl and if not bind the default. Seems ugly ... and I like the fact that since we did this dev, we provide a default binding for Ehcache.
Though I could add a module not enabled that would provide a no-op default. Would then be just about documenting it ...